bucket/filezilla.json

```
@@ -15,9 +15,31 @@
     },
     "extract_dir": "FileZilla-3.47.1",
     "pre_install": [
-        "$file = 'fzdefaults.xml'",
-        "if (-not (Test-Path \"$persist_dir\\$file\")) {",
-        "    Copy-Item \"$dir\\docs\\fzdefaults.xml.example\" \"$dir\\fzdefaults.xml\"",
+        "@'",
+        "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\" ?>",
+        "<FileZilla3>",
+        "  <Settings>",
+        "    <Setting name=\"Config Location\">config</Setting>",
+        "    <Setting name=\"Disable update check\">1</Setting>",
+        "  </Settings>",
+        "</FileZilla3>",
+        "'@ | Out-File \"$dir\\fzdefaults.xml\" -Encoding Ascii",
+        "",
+        "$oldfzdefault = \"$persist_dir\\fzdefaults.xml\"",
+        "if (Test-Path \"$oldfzdefault\") {",
+        "    Write-Host 'Fixing existing configuration...' -f Yellow",
+        "    $oldConfigDir = \"$((Get-Item \"$persist_dir\").PSDrive.Root)filezilla\"",
+        "    $newConfigDir = \"$persist_dir\\config\"",
+        "    if (Test-Path \"$oldConfigDir\") {",
+        "        Write-Host \"Moving existing configuration from '$oldConfigDir' to '$newConfigDir'\" -f Yellow",
+        "        Move-Item \"$oldConfigDir\" \"$newConfigDir\"",
```

This will move folder and put it in folder.

Resulting in `..persist\filezilla\config\filezilla\<configuration files>`

```suggestion
        "        Move-Item \"$oldConfigDir\\*\" \"$newConfigDir\"",
```

---

The destination directory should not exist, in that case the move is correct (i tested it...)
The proposed fix don't work since this destination directory doesn't exists.
In case the destination directory already exists (eg a user has created it manually the 'config' by it self, whatever the reason) i will add a check to handle that